Products/ZenUtils/zenpack.py

```diff
@@ -479,6 +479,17 @@ def doRestore(pack):
         sortedPacks = toposort_flatten(zpsToSort)
         triedReversing = False
         fixedSomething = len(sortedPacks) > 0
+        # Installing all zenpacks with '--files-only' flag
+        # for preventing DistributionNotFound Error
+        for pack in sortedPacks:
+            candidate = self._findEggs(pack, zpsToRestore[pack][0])
+            try:
+                EggPackCmd.InstallEggAndZenPack(
+                    self.dmd,
+                    candidate[0],
+                    filesOnly=True)
```

Comment:
It's OK to have all three arguments on the same line.  It should still be readable.  E.g.
```
    EggPackCmd.InstallEggAndZenPack(
        self.dmd, candidate[0], filesOnly=True)
```